### DISCUSSION TITLE
USB BT dongles for OTA flashing LYWSD03MMC

### DISCUSSION THREAD

**Discussion by mercs759** [pvvx/ATC_MiThermometer#334] [Q&A]
Hi , 
i'm about to buy a BT USB dongle to flash via OTA my LYWSD03 
I see realtek RTL8761B is the only choice for BT5.0 USB dongles under Linux: it's used by almost all 5.0/5.1 dongles.
Some manufacturers advertise their RTL8761B-based dongles as USB2.0

BUT
realtek docs say that RTL8761B works on a usb1.1 bus
and u say "UART speeds of 921600 baud and even USB1.1 are not enough to transfer information in binary form in the HCI format. Requires BT adapter with USB2.0 HS."

So, this chipset will give troubles during OTA flashing? I'm confused

EDIT: tested a Qualcomm "CSR8515 A10" based USBdongle (BT4.0) : dongle is advertised as USB 2.0, but once plugged in PC , Device Monitoring Studio says "full speed". 
Later I found the official CSR doc for csr8515 A10, and confirm this : "Full-speed (12Mbps) USB 2.0 interface"

So, it seems difficult to find a BTUSB dongle at HighSpeed...

I solved using my phone ,Pixel 3a. Flashing went ok.
 
I'd suggest leaving USB dongles out of this.  ;D
Thnx for your job, 
greetings from Italy.

**Comment by rsilves** [comment]
Hi there, yea, +1 on using  phones or tablets (I used Pixel 6a, Samsung Galaxy A8 tablets to flash my LYWSD03MMC successfully).
